### ISSUE TITLE
A link in tensorflow's website is ineffective

### ISSUE THREAD

**Issue by zhmz90** [tensorflow/tensorflow#3403]
The link in **Download [the tutorial code](https://github.com/tensorflow/tensorflow/tree/r0.9/tensorflow/examples/learn/wide_n_deep_tutorial.py).**  from [TensorFlow Linear Model Tutorial](https://www.tensorflow.org/versions/master/tutorials/wide/index.html) failed.

Is the source code of the website not open? I didn't find it.

**Comment by vrv** [comment]
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/g3doc/tutorials/wide Feel free to send a PR to update the docs for us!

**Comment by zhmz90** [comment]
This fixed by a recent commit ac482df8f3522656d9bab96f665405aa2ba5f98b, and the official site doesn't update now. 

```shell
commit ac482df8f3522656d9bab96f665405aa2ba5f98b
Author: A. Unique TensorFlower <gardener@tensorflow.org>
Date:   Thu Jul 7 15:07:26 2016 -0800

    Points links to the wide_n_deep.py example to master, and fixes a heading problem in the tf.contrib.learn quickstart.
    Change: 126857823

diff --git a/tensorflow/g3doc/tutorials/wide_and_deep/index.md b/tensorflow/g3doc/tutorials/wide_and_deep/index.md
index 910e91e..d238b72 100644
--- a/tensorflow/g3doc/tutorials/wide_and_deep/index.md
+++ b/tensorflow/g3doc/tutorials/wide_and_deep/index.md
@@ -42,7 +42,7 @@ To try the code for this tutorial:
 already.
 
 2.  Download [the tutorial code](
-https://www.tensorflow.org/code/tensorflow/examples/learn/wide_n_deep_tutorial.py).
+https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/learn/wide_n_deep_tutorial.py).
 
 3.  Install the pandas data analysis library. tf.learn doesn't require pandas, but it does support it, and this tutorial u
     1. Get `pip`:

```

**Comment by terrytangyuan** [comment]
@vrv The g3doc needs to be regenerated. I saw some other places with this issue as well.
